decoimpact/business/entities/rule_based_model.py

```diff
@@ -28,12 +28,15 @@
 class RuleBasedModel(IModel):
     """Model class for models based on rules"""
 
+    # pylint: disable=too-many-arguments
+    # pylint: disable=too-many-instance-attributes
     def __init__(
         self,
         input_datasets: List[_xr.Dataset],
         rules: List[IRule],
         mapping: Optional[dict[str, str]] = None,
         name: str = "Rule-Based model",
+        partition: str = "",
```

Comment:
is this necessary only for logging? why not just report the file name? (this change is meant for partitions, but you might as well use it for several files that are not partitions?)

Reply:
needed in logger to show which partition is being processed
also needed to create the new outputfilenames.